apps/web-mzima-client/src/app/shared/components/collections/collections.component.html

```diff
@@ -93,6 +93,10 @@
         </app-collection-item>
       </div>
     </div>
+
+    <mat-dialog-actions align="end">
+      <mzima-client-button (buttonClick)="closeModal()"> Done </mzima-client-button>
+    </mat-dialog-actions>
   </ng-container>
```

Comment:
Is this change connected to this issue (https://github.com/ushahidi/platform/issues/4751)?

Reply:
no this isn't related, i forgot to checkout to a new branch. this change is related to the task on adding posts to a collection. I made a PR for it. @Angamanga